index.html

```diff
@@ -287,6 +284,54 @@ <h2>Contact me</h2>
         <div class="line"></div>
       </div>
     </section>
+      <div class="grid g2 popup pop-1">
+        <div class="exit">
+          <span class="bar2"></span>
+          <span class="bar2"></span>
+        </div>
+        <div class="box2">
+          <h1>Tonic</h1>
+          <div class="canopy">
+            <p class="p1">CANOPY</p>
+            <p class="p2">Back End Dev</p>
+            <p class="p3">2015</p>
+          </div>
+          <img class="image1" src="/images/Snapshoot-Portfolio.png" alt="Tonic">
+          <div class="box1">
+            <div class="textSpace">
+            <h2>
+             Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptas, doloribus, porro repellendus autem ducimus
+              aut quos vero officia ut, eligendi consequuntur quam? Aut quae animi maxime modi! Perspiciatis, dolorum laboriosam.
+            </h2>
+            <h2>
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptas, doloribus, porro repellendus autem ducimus
+               aut quos vero officia ut, eligendi consequuntur quam? Aut quae animi maxime modi! Perspiciatis, dolorum laboriosam.
+               
+             </h2>
+            </div>
+            <div class="buttons button-splash">
+              <ul class="list500">
+                <li><button>html</button></li>
+                <li><button>css</button></li>
+                <li><button>javascript</button></li>
+                <li><button>ruby</button></li>
+                <li><button>bootstrap</button></li>
+              </ul>
+              <span class="raya500"></span>
+              <div class="a500">
+                <div class="boton500">
+                  <a>See live <i class="fa-brands fa-creative-commons-share"></i></a>
```

Comment:
- [ ] As said by the previous reviewer, please make sure you use the same image icon as per the original design, you just did not well check the icon because it exists
<img width="356" alt="1" src="https://user-images.githubusercontent.com/77843050/231311257-41fa3404-bac1-4dee-a22d-a1eb9bf453fe.png">